website/server/models/user/hooks.js

```diff
@@ -13,6 +13,7 @@ schema.plugin(baseModel, {
   private: ['auth.local.hashed_password', 'auth.local.salt', '_cronSignature'],
   toJSONTransform: function userToJSON (plainObj, originalDoc) {
     plainObj._tmp = originalDoc._tmp; // be sure to send down drop notifs
+    plainObj.filters = undefined;
```

Comment:
@velezsarain yes, in the client `user.filters` doesn't exists anyway because JSON doesn't have an `undefined` type and so it gets discarded anyway